templates/webapps/galaxy/workflow/run.mako

```diff
@@ -583,9 +583,10 @@ if wf_parms:
                     <%
                     pja_ss_all = []
                     for pja_ss in [ActionBox.get_short_str(pja) for pja in step.post_job_actions]:
-                        pja_ss = h.escape( pja_ss )
                         for rematch in re.findall('\$\{.+?\}', pja_ss):
```

Comment:
Now that I poke at it -- in your examples you're not trusting tool.output_name.  Intentional?  If so, makes me wonder where else we're either trusting it and shouldn't be, or where it's being unnecessarily escaped.